.gitignore

```diff
@@ -1,5 +1,5 @@
 # React Native
-node_modules/
+/node_modules/
```

Comment:
I didn't understand this change until after looking at the diff. Do you think adding an explanation would be useful for future devs?

```suggestion
# Only ignore the node_modules folder that is in the root of the project (via the leading /)
# We have other node_modules folders that we do not wish to ignore
/node_modules/
```

However, I do question whether we need to track those assets in the repository. After all, they are copied from the library at build time (`make bundle-ios`). Tracking binaries in the repo is bound to increase  its size in an inefficient way, since Git can't diff them and therefore only track what changed.

My assumption that it's safe to ignore these files can be verified by deleting them locally, running `make bundle-ios`, and ensuring that the files have been recreated without alteration and that the diff is the same as it was before.

<img width="1648" alt="Screenshot 2023-07-27 at 2 09 55 pm" src="https://github.com/woocommerce/WooCommerce-Shared/assets/1218433/6e6363c0-baca-4085-a74a-7e711f0c6355">

Reply:
One concern with this approach might be that a on a fresh install Xcode won't build out of the box. 

We could add a build phase that ensures the `assets/` folder has children and, if not, fails the build suggesting to run `make bundle-ios` (or running it itself)

Reply:
Nice! I'll add the extra comment for now and will take a look at ignoring the assets + the build phase in a later PR.
In general, I'm not very worried about this scenario because I don't think it will be common to have much more libs that include assets 😅

Reply:
> In general, I'm not very worried about this scenario because I don't think it will be common to have much more libs that include assets 😅

Fair enough. Still, it's unnecessary weight we're carrying around in the repo. I guess we'll see later on if new images arrive or if for some reason these change.